README.md

```diff
@@ -1,34 +1,53 @@
-# RD
+# Rancher Desktop
 
-Note: RD only works on mac at the moment.
+Rancher Desktop is an open-source project to bring Kubernetes and container management to the desktop.
+Windows and MacOS versions of Rancher Desktop are available for download.
 
-This is in pre-alpha and needs to be run using developer tools.
+## Features
 
-## Prerequisites
+Rancher Desktop provides the following features in the form of a desktop application:
 
-* Be on macos (note, expansion to other operating systems is planned)
-* Node.js
+- The version of Kubernetes you choose
+- Ability to test upgrading Kubernetes to a new version and see how your workloads respond
+- Build, push, and pull images (powered by [KIM](https://github.com/rancher/kim))
+- Expose an application in Kubernetes for local access
 
-* The following pre-requisites are needed by the vue testing framework.
+All of this is wrapped in an open-source application.
 
-* macos:
+## Get The App
+
+You can download the application for MacOS and Windows on the [releases page](https://github.com/rancher-sandbox/rd/releases).
+
+Note, [development builds](https://github.com/rancher-sandbox/rd/actions/workflows/package.yaml) are available from the CI system. Development builds are not signed.
+
+## Base Design Details
+
+Rancher Desktop is an electron application with the primary business logic being written in TypeScript and JavaScript. It leverages several other pieces of technology to provide the platform elements which include k3s, kim, kubectl, wsl, hyperkit, and more. The application wraps numerous pieces of technology to provide one cohesive application.
+
+## Building The Source
+
+Rancher can be build from source on MacOS or Windows. The following provides some detail on building.
+
+### Prerequisites
+
+Rancher Desktop is an electron and [node.js](nodejs.org/) application. node.js needs to be installed to build the source.
```

Comment:
Broken link because "/nodejs.org" is interpreted to be relative to the current page. Should be `https://nodejs.org`

Reply:
Also, `electron` should be `[electron](https://www.electronjs.org/)`

Reply:
Fixed